### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by chrisboban**
1. #315 clarified with a note.
2. Link for downloading `.exe` and `.AppImage` updated to the latest version of release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,14 +46,15 @@ guiscrcpy is a multiplatform, ready-to-use GUI layer for Android to PC screen mi
...
-| AppImage      | <img src="https://guiscrcpy.github.io/img/linux.png" height=15px> | [![AppImage](https://github.com/srevinsaju/guiscrcpy/workflows/AppImage/badge.svg)](https://github.com/srevinsaju/guiscrcpy/actions?query=event%3Apush+branch%3Amaster+is%3Asuccess+workflow%3AAppImage+) | [Download `.AppImage`](https://github.com/srevinsaju/guiscrcpy/releases/tag/v4.11.3) |
+| Windows (exe) | <img src="https://guiscrcpy.github.io/img/windows.png" height=15px> | [![Windows Executable](https://github.com/srevinsaju/guiscrcpy/workflows/Windows%20Executable/badge.svg)](https://github.com/srevinsaju/guiscrcpy/actions?query=+event%3Apush++is%3Asuccess+branch%3Amaster+workflow%3A%22Windows+Executable%22) | [Download `.exe`](https://github.com/srevinsaju/guiscrcpy/releases/tag/v2022.7.1)   |
```
Comment (srevinsaju) [inline README.md:56]: when clicking the link on the status page, it leads to an old page. The new page should replace `master` with `main`. 

For example: 
```
https://github.com/srevinsaju/guiscrcpy/actions?query=+event%3Apush++is%3Asuccess+branch%3Amaster+workflow%3A%22Windows+Executable%22
```
should be changed to 
```
https://github.com/srevinsaju/guiscrcpy/actions?query=+event%3Apush++is%3Asuccess+branch%3Amain+workflow%3A%22Windows+Executable%22
```
The same should be done for AppImage row as well.

Comment (srevinsaju) [reply README.md]: ok, my bad. Replace both AppImage's and Windows status badges with this block
```md
[![Continuous](https://github.com/srevinsaju/guiscrcpy/actions/workflows/continuous.yml/badge.svg)](https://github.com/srevinsaju/guiscrcpy/actions/workflows/continuous.yml)
```

Comment (chrisboban) [reply README.md]: image

Hey, may I know if this is the expected output

Comment (srevinsaju) [reply README.md]: yes. that's the correct output.

**Pull request closed without merging**